Detectors/EMCAL/workflow/src/emc-reco-workflow.cxx

```diff
@@ -53,6 +53,7 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
     {"configKeyValues", o2::framework::VariantType::String, "", {"Semicolon separated key=value strings"}},
     {"disable-mc", o2::framework::VariantType::Bool, false, {"disable sending of MC information"}},
     {"disable-decoding-errors", o2::framework::VariantType::Bool, false, {"disable propagating decoding errors"}},
+    {"no-loadrecoparamfromccdb", o2::framework::VariantType::Bool, false, {"disable loading RecoParam object from CCDB"}},
```

Comment:
Do you really need this option? Note that the values passed via `--configKeyValues ...` will override corresponding data-members in the object loaded from the CCDB.

Reply:
We have tested this already for the calib params (which also have a no-load... option). The plan is indeed to fully rely on the CCDB, the option was just a fallback for testing. I can remove it again, will simplify a bit the code.

Reply:
I removed the option to disable loading from the CCDB, indeed I agree it is not really needed.